test/random-battles/all-gens.js

```diff
@@ -21,6 +21,7 @@ describe('value rule support', () => {
 
 	for (let gen = 1; gen <= 8; gen++) {
```

Comment:
shouldn't this be `gen <= 9`?

Reply:
The test would need to be changed for gen 9, since each Pokemon's different sets can have a different number of moves in their movepool.

Reply:
fair